compose-readme.md

```diff
@@ -0,0 +1,175 @@
+# Running Containers Guide
+
+This guide explains how to start, stop, restart, and manage containers in your environment, with tailored instructions for both macOS and Windows users.
+
+## Update submodules (dotnesh-mesh-client)
+
+1. **Update git submodules**
+   The `src/Shared/dotnet-mesh-client` folder needs to be populated if you just cloned the repository. Run this command in the root of the repository (you only need to do this once). Afterward, the `src/Shared/dotnet-mesh-client` will be populated.
+
+   ```bash
+   git submodule update --init --recursive
+   ```
+
+## Setting Up the .env File
+
+Before starting the containers, you need to create a `.env` file based on the provided `env.template` file.
+
+1. **Locate the Template**
+   The `env.template` file is an example file that contains environment variable placeholders.
+
+2. **Create the .env File**
+   Copy the `env.template` file to a new file named `.env`:
+
+   ```bash
+   cp env.template .env
+   ```
+
+3. **Configure the .env File**
+   Open the `.env` file and fill in the required values, such as `DB_NAME`, `PASSWORD`, and `DB_CONNECTION`. Example:
+
+   ```env
+   DB_NAME=ServiceInsightsDB
+   PASSWORD=YourSecurePassword
+   DB_CONNECTION=sql-database  # macOS users
+   #DB_CONNECTION=127.0.0.1  # Windows users
+   ```
+
+   Ensure the correct database connection details are set based on your operating system.
+
+## Starting Containers
+
+There are two options for starting your containers: using a provided shell script or manually running the Podman Compose commands. Choose the method that works best for your workflow.
+
+### Option 1: Using the Shell Script
+
+For simplicity there is a script that will run all the `podman compose` commands in the correct order, based on your operating system.
+
+1. **Run the Script**
+   The script automatically detects your operating system (macOS or Windows via WSL) and starts the necessary containers.
+
+   ```bash
+   ./run-containers.sh
```

Comment:
If you move the script you'll need to update this

Reply:
ok updated :)